src/EnergyPlus/StateManagement.cc

```diff
@@ -216,12 +216,11 @@
 #include <EnergyPlus/WaterToAirHeatPumpSimple.hh>
 #include <EnergyPlus/WeatherManager.hh>
 
-void EnergyPlus::clearThisState(EnergyPlusData &state)
+void EnergyPlus::clearAllStates(EnergyPlusData &state)
```

Comment:
This was a really awkward situation, and it is now much better.  Now the clear all states function must take in a state argument.  The clear all states function will now clear the state of the argument, then it will go through and clear the remaining global states as well.  As the global refactor gets done, the number of "extra" clear_state calls in here will reduce to the point where this is just a call to `state.clear_state()`, and this function can be eliminated.